sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -247,7 +247,28 @@ case class DropTableCommand(
     } else if (ifExists) {
       // no-op
     } else {
-      throw QueryCompilationErrors.tableOrViewNotFoundError(tableName.identifier)
+      throw QueryCompilationErrors.noSuchTableError(
```

Comment:
`DropTableCommand` now won't be used to drop temp view, right? If so, there is some logic around ` val isTempView = catalog.isTempView(tableName)`, do we need update it?

Reply:
good point, we can simplify v1 DROP TABLE command now